test_scripts/Polices/External_UCS_Informing_HMI/001_ATF_P_GetListofPermissions_with_appID_all_ecs_in_bound_EntitySatus_ON.lua

```diff
@@ -0,0 +1,127 @@
+---------------------------------------------------------------------------------------------
```

Comment:
@maria-mih, please clarify with BA should SDL send OnAppPermissionChanged when:

Precondition:
1. Application is listed in sdl_preloaded_pt.json, device is not consented. 
2. Application has assigned groups that require user consent.
3. Start SDL and HMI, 
4. Register the application(it is listed in preloaded file.)
5. Activate application.
Step:
1. PTU is triggered to get consent of device. ("isPermissionsConsentNeeded":true)

Question: Should SDL send OnAppPermissionChanged to request user consent OR
OnAppPermissionChanged should be sent only after PTU.
I didn't manage to find applicable requirement for this situation.

Note: if SDL should send OnAppPermissionChanged after device consent, script should be updated .